Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.Tests/Lib/Helpers.cs

```diff
@@ -37,7 +37,7 @@ internal static void DeleteSourceDir()
     {
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/56824f42a881d74709954eeb239fb474ee075540/Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.Tests/Lib/Helpers.cs#L17
All of these properties can be `readonly`.